text_generation/causal_lm/cpp/continuous_batching/library/src/sequence_group.hpp

```diff
@@ -112,7 +112,8 @@ class Sequence {
     }
 
     float get_cumulative_log_probs() const {
-        return m_cumulative_log_prob;
+        float cumulative_log_prob = std::accumulate(m_log_probs.begin(), m_log_probs.end(), 0.f);
+        return cumulative_log_prob;
```

Comment:
do we have checks that this value aligned with HF?

Reply:
Moved to open. Need to be checked